jsconfig.json

```diff
@@ -1,14 +1,3 @@
 {
-  "exclude": [
```

Comment:
What prompted this? :eyes:

Reply:
i keep removing all of this stuff cause its killing autocomplete. I'll remove it for this pr and figure out a proper fix.

Reply:
removed